app/controllers/posts_controller.rb

```diff
@@ -9,7 +9,3 @@ def show
     @post = @user.posts.find(params[:id])
   end
 end
```

Comment:
- [ ] there is no create method in your postcontroller, so I am not able to create a post, kindly add this
![Screenshot from 2022-12-10 22-28-27](https://user-images.githubusercontent.com/66097371/206875970-d5ccff7e-9cac-41f1-b068-693571baa2d7.png)